packages/core/src/utils.ts

```diff
@@ -663,11 +665,15 @@ export const prettyFunctionCall = (
     removedBrackets + ' '.repeat(numSpacesForClosingParenthesis)
 
   const addressTag = address !== '' ? `<${address}>` : ''
+  const valueString =
+    BigInt(value) > BigInt(0)
+      ? `{ value: ${formatUnits(BigInt(value), 'ether')} ether }`
```

Comment:
I don't see any actionable solution to this. If we render everything as wei, then the user still has to count zeros. We could [render all of the possible units](https://www.geeksforgeeks.org/what-are-the-different-units-used-in-ethereum/), but frankly, I don't see value in that because people don't know them.